app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import { Formik, Form as FormikForm } from "formik";
+import { Button, Pane } from "neetoui";
+import { Input, Select } from "neetoui/formik";
+
+import formValidationSchemas from "constants/formValidationSchemas";
+
+import { ROLE_OPTIONS } from "../constants";
+
+const Form = ({ onClose, contact, handleSubmit }) => {
+  return (
+    <Formik
+      initialValues={contact}
+      onSubmit={handleSubmit}
+      validationSchema={formValidationSchemas.contactsForm}
+    >
+      {({ isSubmitting }) => (
+        <FormikForm className="w-full">
+          <Pane.Body>
+            <div className="space-y-8 w-full">
```

Comment:
Is there a need for this `div`? What's the issue with passing the className along with the `Pane.Body`?

Reply:
Without this div, I will have to pass "w-full" to all of the child components.

Reply:
@vijay2136 The question was on why we are not able to pass this same `className` attr to `Pane.Body`(which is the parent for all the children) and avoid having this `div`.

Reply:
https://vimeo.com/670721598/64d9e2df9d

Reply:
@vijay2136 

You still haven't addressed this part:

> The question was on why we are not able to pass this same className attr to Pane.Body

```jsx
<Pane.Body className="space-y-8 w-full">
{children without any wrapping div as parent}
</Pane.Body>
```

Please make a video and show me how the UI behaves in this case.